doc/sphinxext/math_symbol_table.py

```diff
@@ -73,6 +58,27 @@
 ]
 
 
+def calculate_best_columns(
+        symbols_list, max_line_length, max_columns
+):
+    """
+    Calculate the best number of columns to fit the symbols within the
+    given constraints.
+
+    Parameters
+    ----------
+        symbols_list (list): A list of symbols to be displayed.
+        max_line_length (int): The maximum allowed length of a line.
+        max_columns (int): The maximum number of columns to consider.
+
+    Returns
+    -------
+        int: The best number of columns that fits within the constraints.
```

Comment:
Docstring layouts should follow the [numpydoc style](https://numpydoc.readthedocs.io/en/latest/format.html#parameters).